.gitignore

```diff
@@ -128,8 +127,6 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-CarShowRoom/Pipfile
-env
 .idea
 CarShowRoom/.idea
-
+env/db.env
```

Comment:
nitpick: env/.env
So you don’t need to include other env files in case you will create any in the future